sitewit/constants.py

```diff
@@ -2,6 +2,17 @@ class CampaignServiceTypes(object):
     QUICKSTART = 'quickstart'
 
 
+class BillingTypes(object):
+    AUTOMATIC = 'Automatic'
+    TRIGGERED = 'Triggered'
+
+
+class CampaignTypes(object):
+    DISPLAY = 'display'
+    SEARCH = 'search'
+
+
 CAMPAIGN_SERVICES = {
     CampaignServiceTypes.QUICKSTART: 'QuickStart Campaign'
 }
+SPEND_CHARGE_ITEM_TYPES = ('DisplaySpend', 'SearchCampaignSpend')
```

Comment:
missing blank line?

Reply:
I dunno whether we should add a blank line there... Let me check.

Reply:
Didn't find anything in PEP8 regarding that.